<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcmini</class>
 <widget class="QWidget" name="calcmini">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <widget class="QPushButton" name="btnAdicao">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubtracao">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultiplicacao">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDivisao">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrimeiroNumero">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSegundoNumero">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtResultado">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
